src/presets/semver-workflow.ts

```diff
@@ -1,3 +1,5 @@
+// Node 14 does not implement String.replaceAll: https://node.green/#ES2021-features--String-prototype-replaceAll
```

Comment:
Using replaceAll is just me being a silly Javaboy. Lets rewrite to use replace. I'll fix on this branch.